health/notifications/pagerduty/README.md

```diff
@@ -1,46 +1,67 @@
 <!--
-title: "PagerDuty"
+title: "Send alert notifications to PagerDuty"
+description: "Send alerts to your PagerDuty dashboard any time an anomaly or performance issue strikes a node in your infrastructure."
+sidebar_label: "PagerDuty"
 custom_edit_url: https://github.com/netdata/netdata/edit/master/health/notifications/pagerduty/README.md
 -->
 
-# PagerDuty
+# Send alert notifications to PagerDuty
 
-[PagerDuty](https://www.pagerduty.com/company/) is the enterprise incident resolution service that integrates with ITOps and DevOps monitoring stacks to improve operational reliability and agility. From enriching and aggregating events to correlating them into incidents, PagerDuty streamlines the incident management process by reducing alert noise and resolution times.
+[PagerDuty](https://www.pagerduty.com/company/) is an enterprise incident resolution service that integrates with ITOps
+and DevOps monitoring stacks to improve operational reliability and agility. From enriching and aggregating events to
+correlating them into incidents, PagerDuty streamlines the incident management process by reducing alert noise and
+resolution times.
 
-Here is an example of a PagerDuty dashboard with Netdata notifications:
+Here's an example of a PagerDuty dashboard with Netdata alert notifications:
 
-![PagerDuty dashboard with Netdata notifications](https://cloud.githubusercontent.com/assets/19278582/21233877/b466a08a-c2a5-11e6-8d66-ee6eed43818f.png)
+![PagerDuty dashboard with Netdata alert notifications](https://user-images.githubusercontent.com/1153921/118317133-872a4100-b4ac-11eb-9cf1-70414aba010f.png)
```

Comment:
[The screenshot](https://user-images.githubusercontent.com/1153921/118317133-872a4100-b4ac-11eb-9cf1-70414aba010f.png) is [very blurry](https://i.kym-cdn.com/photos/images/newsfeed/002/117/476/261.jpg).

Reply:
What do we do? @kickoke 

- close this PR.
- remove the screenshot, merge the PR.
- keep the screenshot, merge the PR.

Reply:
I would remove the screenshot and then merge the PR. 
We struggle with maintaining our own screenshots, so I don't want to add maintaining 3rd party screenshots to the list.
Thank you @ilyam8